Lib/fontTools/ttLib/ttFont.py

```diff
@@ -89,7 +91,7 @@ def __init__(self, file=None, res_name_or_index=None,
 			sfntVersion="\000\001\000\000", flavor=None, checkChecksums=0,
 			verbose=None, recalcBBoxes=True, allowVID=NotImplemented, ignoreDecompileErrors=False,
 			recalcTimestamp=True, fontNumber=-1, lazy=None, quiet=None,
-			_tableCache=None):
+			_tableCache=None, cfg={}):
```

Comment:
in general mutable default arguments are bad, but in this case you always make a copy so it should be fine. Alternatively you default to `cfg=None` and then in method body do:

```
if cfg is None:
    cfg = Config()
self.cfg = cfg.copy() if isinstance(cfg, AbstractConfig) else Config(cfg)
```

Reply:
actually it's fine like this